src/clj/puppetlabs/puppetserver/cli/subcommand.clj

```diff
@@ -52,3 +52,11 @@
       (throw+))
     (finally
       (shutdown-agents))))
+
+(defn jruby-run
```

Comment:
Would it be better to push this behavior into https://github.com/puppetlabs/jruby-utils/blob/master/src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj#L279 ? Or is it better to keep the invocation of `System/exit` as far down the stack as possible?